.github/workflows/ci.yml

```diff
@@ -11,7 +11,7 @@ jobs:
       matrix:
         # Windows on macOS builds started failing, so they are disabled for noew
 
-        ruby: [2.4, 2.5, 2.6, 2.7]
+        ruby: ['2.4', '2.5', '2.6', '2.7', '3.0']
```

Comment:
Feel free to drop some older versions here.

Reply:
As long as they don't cause maintenance pain, I'm fine to keep 2.4.